settings.h

```diff
@@ -802,6 +802,7 @@ typedef enum {
     ToolChange_Manual,
     ToolChange_Manual_G59_3,
     ToolChange_SemiAutomatic,
+    ToolChange_FastSemiAutomatic,
```

Comment:
ToolChange_FastSemiAutomatic must be after ToolChange_Ignore to match the order of options in the list in line 2100. And the leading space in the new option in line 2100 may cause layout misalignment in senders.